bin/cache.js

```diff
@@ -0,0 +1,12 @@
+const Catbox = require('catbox');
```

Comment:
@Danwhy this is good. 
would you mind adding a simple example of using catbox to: [`learn-hapi`](https://github.com/dwyl/learn-hapi/issues/48) _please/thanks_!